### DISCUSSION TITLE
Integrating the Alchemy Account Kit SDK into game engines like Unity and UnealEngine

### DISCUSSION THREAD

**Discussion by chista0x** [alchemyplatform/aa-sdk#833] [Ideas]
Hi,

I'm interested in integrating the Alchemy Account Kit SDK into game engines like Unity (C#) and Unreal Engine (C++). Does your SDK support these environments directly, or are there recommended practices for bridging TypeScript with these languages? Any guidance or examples would be greatly appreciated. Additionally, are there any grants or support available for developers looking to implement this in game engines?

Thank you!

**Comment by avarobinson** [comment]
Hi @chista0x.

Excited to hear you're building with account kit! We currently do not support these environments directly. Feel free to adapt our open-source sdk to your needs or use API methods directly for [bundler](https://docs.alchemy.com/reference/bundler-api-quickstart) and [gas manager](https://docs.alchemy.com/reference/gas-manager-coverage-api-quickstart) support.
